### ISSUE TITLE
[Andrew] Searching Feature based on date, title etc

### ISSUE THREAD

**Issue by AndrewPhilipJoseph** [domph/p1-rain#22]

**Comment by AlexH04** [comment]
Sounds doable, nice application of search algorithms

**Comment by AdityaS1426** [comment]
I think your plan for this week seems great. The goals are very realistic as well. I would suggest adding a description to this ticket in order to know your goals for this week.

**Comment by andrewcomputsci2019** [comment]
I agree with Aditya you should add checkboxes under your description to better mark what needs to be accomplished.
